package.json

```diff
@@ -19,9 +19,9 @@
     "lint:js": "eslint . .storybook --ext .js,.jsx",
     "lint:js:fix": "eslint . .storybook --ext .js,.jsx --fix",
     "lint": "npm run lint:js && npm run lint:scss",
-    "jest:update": "TZ=Europe/London jest --updateSnapshot",
-    "jest:watch": "TZ=Europe/London jest --watch",
-    "jest": "TZ=Europe/London jest --coverage",
+    "jest:update": "TZ=Etc/UTC jest --updateSnapshot",
+    "jest:watch": "TZ=Etc/UTC jest --watch",
+    "jest": "TZ=Etc/UTC jest --coverage",
```

Comment:
One of the snapshots changed to use BST. I expect this is because something internally changed in Jest, so I set the timezone to UTC here.